not-to-release/sources/reviews/034813.xml.conllu

```diff
@@ -42,7 +42,7 @@
 20	and	and	CCONJ	CC	_	25	cc	25:cc	_
 21	especially	especially	ADV	RB	_	25	advmod	25:advmod	_
 22	their	they	PRON	PRP$	Number=Plur|Person=3|Poss=Yes|PronType=Prs	25	nmod:poss	25:nmod:poss	_
-23	signature	signature	NOUN	NN	Number=Sing	25	compound	25:compound	_
```

Comment:
This seems like a compound to me. Cf. * _The signature and delicious banana split_*. If it were an adjective, you should be able to form a conjunction with another `ADJ`.

Reply:
A couple of adjective-like attestations in COCA:

* rodent activity was observed in both signature and regular tent cabins
* by the glass starting at $6 and a selection of signature and classic cocktails

These sound perfectly fine to me, though coordination with an adjective is probably rarer than for *custom*.

*_signature and delicious banana split_ definitely sounds odd, but perhaps there's a semantic explanation? And/or maybe it matters whether it has a possessive modifier: *X's signature Y* means something like 'the Y that is X's hallmark', but I don't know if that meaning is quite the same as in the COCA example above.

http://www.dictionary.com/browse/signature lists it as an adjective.

Reply:
Dictionaries often have very strange definitions of POS tags, so I won't count that as evidence :)

Re the COCA examples: these could in my opinion be still seen as examples of compounding with an elided noun. Note that you could also say _rodent activity was observed in both the signature and the regular tent cabins_ which suggests that this might actually be a conjoined NP with an elided head noun and not a conjoined adjectival phrase. 

What is your motivation for changing the POS tag?

Reply:
Admittedly, that would also work for the _banana split_ example but that would change the meaning.

Reply:
I think I first convinced myself that *custom* was more like an adjective than a noun, then saw *signature* and figured it was similar. :) But I have less of a strong opinion about *signature*.

Reply:
I wouldn't be surprised if *signature* becomes more adjectivy in the future, but it seems reasonable to call it a noun for now.